nerfstudio/scripts/exporter.py

```diff
@@ -577,8 +579,15 @@ def main(self) -> None:
                 for i in range(shs_rest.shape[-1]):
                     map_to_tensors[f"f_rest_{i}"] = shs_rest[:, i, None]
             else:
-                colors = torch.clamp(model.colors.clone(), 0.0, 1.0).data.cpu().numpy()
-                map_to_tensors["colors"] = (colors * 255).astype(np.uint8)
+                if self.use_sh0_renderer:
```

Comment:
Perhaps a warning should be raised if `use_sh0_renderer` is `True` but the model has sh degree > 0? Since then it gets ignored.

Reply:
Sounds good.